lib/Models/CsvCatalogItem.js

```diff
@@ -174,14 +174,30 @@ CsvCatalogItem.prototype.loadIntoTableStructure = function(url) {
  */
 CsvCatalogItem.prototype.startPolling = function() {
   const polling = this.polling;
+
   if (defined(polling.seconds) && polling.seconds > 0) {
     var item = this;
+
+    // Initialise polling and timer variables, because they might not be set yet
+    polling.isPolling = item.isEnabled;
```

Comment:
A separate `isPolling` variable is not strictly necessary, you can check if there's a nextScheduledUpdateTime. But this way someone using this (albeit internal) API doesn't need to know that nextScheduledUpdateTime = isPolling, which might also not always be the case? Maybe?